sql/testdata/datetime

```diff
@@ -42,6 +42,44 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
+SET TIME ZONE 'EUROPE/ROME'
+
+statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\) violates unique constraint "primary"
```

Comment:
Duplicate key errors are a cumbersome way to test this; it would be easier to do `SELECT '...'::timestamp`.